html-block-and-inline/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>HTML Block and Inline</title>
+</head>
+
+<body>
+  <h1>Blue Americana Snuggie &reg; Blanket</h1>
+  <h2>by SnuggieStore</h2>
+  <div>Price: <del>$19.99</del> $14.99</div>
+  <div> Part of our new Americana collection, this <i>beautiful</i> blue is already a customer favorite!</div>
+  <div>Enjoy complete <strong>Freedom of Movement!</strong></div>
+  <div>Does not slip and slide like a regular blanket and keeps you warm from head to toe.</div>
+
+
+
+</body>
+
+
+
+
+
+
```

Comment:
Please remove these extra lines, there should be no need for extra whitespace in HTML like this.